devguide/examples/map-editFeature.js

```diff
@@ -1,68 +1,55 @@
+var overlay1 = new emp3.api.Overlay({
```

Comment:
This one is so similar to the draw....maybe we should combine this to one example?